### PR TITLE
Prepare 1.1.0

### PR DESCRIPTION
**Pull request by jmaupetit**
Test our release process and containers labeling.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+## 1.1.0 (2017/11/15)
+
+This is our very first release. It has bot been tagged as `1.0.0` as it should
```
Comment (willdurand) [inline CHANGELOG.md]: s/bot/not

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+have, but it took time to have a fully functional deployment strategy.
```
Comment (willdurand) [inline CHANGELOG.md]: should have... been?

Comment (willdurand) [reply CHANGELOG.md]: but => because?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Features:
```
Comment (willdurand) [inline CHANGELOG.md]: Highlighted features?

File: Dockerfile [hunk 4]
```diff
@@ -3,6 +3,20 @@ FROM python:3.6
 # Development build
 ARG IS_NOT_PRODUCTION
 
+# Build-time metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+LABEL org.label-schema.build-date=$BUILD_DATE \
```
Comment (willdurand) [inline Dockerfile]: should we use quotes around variables?

File: Dockerfile [hunk 5]
```diff
@@ -3,6 +3,20 @@ FROM python:3.6
...
+      org.label-schema.name="Pixel" \
+      org.label-schema.description="Integration of smart 'omics' data" \
+      org.label-schema.url="https://pixel.candihub.eu/" \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.vcs-url="https://github.com/Candihub/pixel/" \
+      org.label-schema.vendor="CandiHub" \
+      org.label-schema.version=$VERSION \
+      org.label-schema.schema-version="1.0"
```
Comment (willdurand) [inline Dockerfile]: 😍

File: Dockerfile (hunk 4 above)
Comment (jmaupetit) [reply Dockerfile]: Not sure. I've checked Docker's documentation.

**Pull request merged**